src/ess/reduce/nexus/workflow.py

```diff
@@ -96,6 +100,22 @@ def component_spec_by_name(
     )
 
 
+def disk_chopper_component_spec(
+    filename: NeXusFileSpec[RunType],
+) -> NeXusComponentLocationSpec[snx.NXdisk_chopper, RunType]:
+    """Create a location spec for a disk chopper group in a NeXus file."""
+    return NeXusComponentLocationSpec[snx.NXdisk_chopper, RunType](
+        filename=filename.value
+    )
+
+
+def crystal_component_spec(
+    filename: NeXusFileSpec[RunType],
+) -> NeXusComponentLocationSpec[snx.NXcrystal, RunType]:
+    """Create a location spec for a crystal / analyzer group in a NeXus file."""
+    return NeXusComponentLocationSpec[snx.NXcrystal, RunType](filename=filename.value)
```

Comment:
Feels redundant, since there is no component name, so what is the point in distinguishing between component types? Should we just have `NeXusEntrySpec[RunType]` or something like that, with not component name?